README.md

```diff
@@ -8,6 +8,9 @@ An [InfluxDB](http://influxdb.org/) Node.js Client
 [![code climate](http://img.shields.io/codeclimate/github/node-influx/node-influx.svg?style=flat-square)](https://codeclimate.com/github/node-influx/node-influx)
 [![Dependency Status](https://img.shields.io/david/node-influx/node-influx.svg?style=flat-square)](https://david-dm.org/node-influx/node-influx)
 [![Github Releases](https://img.shields.io/npm/dm/influx.svg?style=flat-square)](https://github.com/node-influx/node-influx)
+[![Gitter](https://img.shields.io/gitter/room/nwjs/nw.js.svg?maxAge=2592000&style=flat-square)](https://gitter.im/node-influx/node-influx?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
```

Comment:
Oh bollocks haha, good catch! Shall update later when I'm in front of the
screen
On 7 May 2016 1:52 pm, "Connor Peet" <notifications@github.com> wrote:

> In README.md
> <https://github.com/node-influx/node-influx/pull/140#discussion_r62416057>
> :
>
> > @@ -8,6 +8,9 @@ An [InfluxDB](http://influxdb.org/) Node.js Client
> >  [![code climate](http://img.shields.io/codeclimate/github/node-influx/node-influx.svg?style=flat-square)](https://codeclimate.com/github/node-influx/node-influx)
> >  [![Dependency Status](https://img.shields.io/david/node-influx/node-influx.svg?style=flat-square)](https://david-dm.org/node-influx/node-influx)
> >  [![Github Releases](https://img.shields.io/npm/dm/influx.svg?style=flat-square)](https://github.com/node-influx/node-influx)
> > +[![Gitter](https://img.shields.io/gitter/room/nwjs/nw.js.svg?maxAge=2592000&style=flat-square)](https://gitter.im/node-influx/node-influx?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
>
> The badge here is for nw.js
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/node-influx/node-influx/pull/140/files/a363af1cda709ddf4417fd60e1bd864bf31f759d#r62416057>
>